atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts

```diff
@@ -131,19 +176,22 @@ export const TitleContainer = styled.div`
   }
 
   h2 {
-    ${fluidRange({ prop: 'fontSize', fromSize: '40px', toSize: '60px' })};
-    ${fluidRange({ prop: 'lineHeight', fromSize: '48px', toSize: '68px' })};
+    ${fluidRange({ prop: 'fontSize', fromSize: '40px', toSize: '72px' }, '320px', '1920px')};
```

Comment:
If possible, let's use a breakpoint from the theme here

Reply:
ah yes, good catch

Reply:
Fixed